dbtmetabase/_models.py

```diff
@@ -100,7 +100,7 @@ def export_models(
 
                     field = table.get("fields", {}).get(column_name)
                     if not field:
-                        if table.get("visibility_type") == "hidden":
+                        if table.get("visibility_type") is not None:
```

Comment:
Not the only instance of "hidden" comparison you need to fix

Reply:
ah sorry - been a busy week - I can see you've merged this anyway so thanks!